metadata-ingestion/src/datahub/ingestion/source/sql/hive/hive_metastore_source.py

```diff
@@ -164,9 +319,43 @@ def get_sql_alchemy_url(
         )
 
 
+# =============================================================================
+# Source Report
+# =============================================================================
+
+
+@dataclasses.dataclass
+class HiveMetastoreSourceReport(StaleEntityRemovalSourceReport):
```

Comment:
If Report and config would be in a separate file it would make it more readable and searchable as well